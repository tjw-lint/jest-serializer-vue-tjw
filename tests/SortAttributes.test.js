const helpers = require('./helpers.js');

import { shallowMount } from '@vue/test-utils';
import SortAttributes from './components/SortAttributes.vue';

describe('SortAttributes.vue', () => {
  test('Sorted', () => {
    helpers.mockSettings({ sortAttributes: true });

    const wrapper = shallowMount(SortAttributes);

    expect(wrapper)
      .toMatchSnapshot();
  });

  test('Unsorted', () => {
    helpers.mockSettings({ sortAttributes: false });

    const wrapper = shallowMount(SortAttributes);

    expect(wrapper)
      .toMatchSnapshot();
  });
});
