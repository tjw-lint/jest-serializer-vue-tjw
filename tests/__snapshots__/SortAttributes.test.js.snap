// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SortAttributes.vue Sorted 1`] = `
<div>
  <h1 class="meow nyan" id="kitten" title="cat">
    fluffy
  </h1>
  <h2 asdf="qwer" id="kitten2">
    mittens
  </h2>
  <h3 asdf id>
    scruffy
  </h3>
</div>
`;

exports[`SortAttributes.vue Unsorted 1`] = `
<div>
  <h1 id="kitten" title="cat" class="meow nyan">
    fluffy
  </h1>
  <h2 id="kitten2" asdf="qwer">
    mittens
  </h2>
  <h3 id asdf>
    scruffy
  </h3>
</div>
`;
